src/content/docs/en/guides/integrations-guide/deno.mdx

```diff
@@ -1,188 +1,13 @@
-
-```js
-import './dist/server/entry.mjs';
-```
-
-See the `start` option below for how you can have more control over starting the Astro server.
-
-You can also run the script directly using deno:
-
-```sh
-deno run --allow-net --allow-read --allow-env ./dist/server/entry.mjs
-```
-
-## Configuration
-
-To configure this adapter, pass an object to the `deno()` function call in `astro.config.mjs`.
-
-```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config';
-import deno from '@astrojs/deno';
-
-export default defineConfig({
-  output: 'server',
-  adapter: deno({
-    //options go here
-  }),
-});
-```
-
-### start
-
-This adapter automatically starts a server when it is imported. You can turn this off with the `start` option:
-
-```js
-import { defineConfig } from 'astro/config';
-import deno from '@astrojs/deno';
-
-export default defineConfig({
-  output: 'server',
-  adapter: deno({
-    start: false,
-  }),
-});
-```
-
-If you disable this, you need to write your own Deno web server. Import and call `handle` from the generated entry script to render requests:
-
-```ts
-import { serve } from 'https://deno.land/std@0.167.0/http/server.ts';
-import { handle } from './dist/server/entry.mjs';
-
-serve((req: Request) => {
-  // Check the request, maybe do static file handling here.
-
-  return handle(req);
-});
-```
-
-### port and hostname
-
-You can set the port (default: `8085`) and hostname (default: `0.0.0.0`) for the deno server to use. If `start` is false, this has no effect; your own server must configure the port and hostname.
-
-```js
-import { defineConfig } from 'astro/config';
-import deno from '@astrojs/deno';
-
-export default defineConfig({
-  output: 'server',
-  adapter: deno({
-    port: 8081,
-    hostname: 'myhost',
-  }),
-});
-```
-
-## Examples
-
-The [Astro Deno](https://github.com/withastro/astro/tree/main/examples/deno) example includes a `preview` command that runs the entry script directly. Run `npm run build` then `npm run preview` to run the production deno server.
-
-## Troubleshooting
-
-For help, check out the `#support` channel on [Discord](https://astro.build/chat). Our friendly Support Squad members are here to help!
-
-You can also check our [Astro Integration Documentation][astro-integration] for more on integrations.
-
-## Contributing
-
-This package is maintained by Astro's Core team. You're welcome to submit an issue or PR!
-
-## Changelog
-
-See [CHANGELOG.md](https://github.com/withastro/astro/tree/main/packages/integrations/deno/CHANGELOG.md) for a history of changes to this integration.
+The Deno adapter allows Astro to deploy your SSR site to Deno targets including Deno Deploy. 
 
-[astro-integration]: /en/guides/integrations-guide/
+The Deno adapter was previously maintained by Astro but now is maintained by Deno directly. Usage is now documented [in the Deno adapter repository](https://github.com/denoland/deno-astro-adapter).
```

Comment:
``````suggestion
The Deno adapter was previously maintained by Astro but now is maintained by Deno directly. Usage is now documented [in the Deno adapter repository](https://github.com/denoland/deno-astro-adapter).

## Migrate to the new adapter

If you are currently using this Astro adapter, you will need to migrate to the new Deno version.
 
1. Remove the existing adapter by [uninstalling the `@astrojs/deno` integration](/en/guides/integrations-guide/#removing-an-integration). 
2. Install `deno-astro-adapter` by following [Deno's installation instructions](https://github.com/denoland/deno-astro-adapter#installation).
3. Update the import statement in `astro.config.mjs` to use the new package:
   ```diff
   - import deno from '@astrojs/deno';
   + import deno from 'deno-astro-adapter';
   ```
``````
BIG NOTE: The instructions on their readme still refer to `@astrojs/deno` and include all the existing installation instructions. That needs to be updated over there!

Reply:
@matthewp What about this as a compromise?